packages/kbn-shared-ux-components/src/empty_state/kibana_no_data_page.tsx

```diff
@@ -17,19 +18,26 @@ export interface Props {
 
 export const KibanaNoDataPage = ({ onDataViewCreated, noDataConfig }: Props) => {
   const { hasESData, hasUserDataView } = useData();
+  const [hasFinishedLoading, setHasFinishedLoading] = useState(false);
```

Comment:
I usually use the `isLoading` flag for this. Then on L38 you don't need a negation. Seems more direct to me.